src/BlackSlope.Api/Startup.cs

```diff
@@ -39,7 +39,11 @@ public void ConfigureServices(IServiceCollection services)
             services.AddAutoMapper(GetAssemblyNamesToScanForMapperProfiles());
             services.AddCorrelation();
             services.AddTransient<IFileSystem, FileSystem>();
-            services.AddTransient<IVersionService, AssemblyVersionService>();
+
+            // TODO: Pick one of the below versioning services

```

Comment:
Is this todo intended for this PR or a separate issue?

Reply:
I'm going to switch this over to `NOTE:`. It's not really an item to fix as much as a "Hey, configure this for a new proj" thing.

Reply:
Done.